### PR TITLE
Translate content/ru/docs/concepts/architecture/controller.md to russian

### PR DESCRIPTION
**Pull request by mshalmanov**
Hello

Translate 
content/ru/docs/concepts/architecture/cloud-controller.md
content/ru/docs/concepts/architecture/controller.md
to russian

### CODE REVIEW

File: content/ru/docs/concepts/architecture/cloud-controller.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+![Kubernetes components](/images/docs/components-of-kubernetes.svg)
+
+Диспетчер облочных контроллеров работает в панели управления как реплицированный набот процессов 
+(обычно это контейнер в Pod-ах). Каждый диспетчер облочных контроллеров реализует многоразовые Each cloud-controller-manager implements
+multiple {{< glossary_tooltip text="контроллеры" term_id="controller" >}} в единственном
+процессе.
+
+
+{{< Примечание: >}}
```
Comment (jihoon-seo) [inline content/ru/docs/concepts/architecture/cloud-controller.md:31]: CI tests fail.

https://app.netlify.com/sites/kubernetes-io-master-staging/deploys/6077c9e13e5b640007c61fac

```
2:07:24 PM: Error: Error building site: "/opt/build/repo/content/ru/docs/concepts/architecture/cloud-controller.md:31:1": failed to extract shortcode: template for shortcode "Примечание" not found
2:07:24 PM: Makefile:27: recipe for target 'build' failed
2:07:24 PM: make: *** [build] Error 255
```

```suggestion
{{< note >}}
```

File: content/ru/docs/concepts/architecture/controller.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+Пока контроллеры вашего кластера работают и могут вносить полезные изменения, не имеет  значения, является ли общее состояние стабильным или нет.
+
+## Дизайн
+
+В качестве принципа своей конструкции Kubernetes использует множество контроллеров, каждый из которых управляет определенным аспектом состояния кластера. Чаще всего конкретный контур управления (контроллер) использует один вид ресурса в качестве своего желаемого состояния и имеет другой вид ресурса, которым он управляет, чтобы это случилось. Например, контроллер для заданий отслеживает объекты заданий (для обнаружения новой работы) и объекты модулей (для выполнения заданий, а затем для того, чтобы видеть, когда работа завершена). В этом случае что-то еще создает задания, тогда как контроллер заданий создает Pod-ы.
+
+Полезно иметь простые контроллеры, а не один монолитный набор взаимосвязанных контуров управления. Контроллеры могут выйти из строя, поэтому Kubernetes предназначен для этого.
+
+{{< Примечание >}}
```
Comment (jihoon-seo) [inline content/ru/docs/concepts/architecture/controller.md:93]:
```suggestion
{{< note >}}
```

**Review by jihoon-seo — CHANGES_REQUESTED** [review]

**Comment by mshalmanov** [comment]
Hello. After edit my translate files, run checks  and show error in the cloud-controller.md file. How i can fix my error ? 

`2:53:43 PM: Error: Error building site: "/opt/build/repo/content/ru/docs/concepts/architecture/cloud-controller.md:13:1": failed to render shortcode "glossary_definition": failed to process shortcode: "/opt/build/repo/layouts/shortcodes/glossary_definition.html:18:19": execute of template failed: template: shortcodes/glossary_definition.html:18:19: executing "shortcodes/glossary_definition.html" at <$term_info.Content>: nil pointer evaluating resource.Resource.Content
2:53:43 PM: Makefile:27: recipe for target 'build' failed`

**Comment by karuppiah7890** [comment]
@mshalmanov There are two errors

```
5:06:09 PM: ERROR 2021/05/25 11:36:09 [ru] "docs/concepts/architecture/cloud-controller.md": "cloud-controller-manager" is not a valid glossary term_id, see ./docs/reference/glossary/* for a full list
ERROR 2021/05/25 11:36:09 [ru] no Resource or Page matching "/static/community-values.md*".
```

The first -

```
5:06:09 PM: ERROR 2021/05/25 11:36:09 [ru] "docs/concepts/architecture/cloud-controller.md": "cloud-controller-manager" is not a valid glossary term_id, see ./docs/reference/glossary/* for a full list
```

This is related to this failure -

```
5:06:09 PM: Error: Error building site: "/opt/build/repo/content/ru/docs/concepts/architecture/cloud-controller.md:13:1": failed to render shortcode "glossary_definition": failed to process shortcode: "/opt/build/repo/layouts/shortcodes/glossary_definition.html:18:19": execute of template failed: template: shortcodes/glossary_definition.html:18:19: executing "shortcodes/glossary_definition.html" at <$term_info.Content>: nil pointer evaluating resource.Resource.Content
```

This is because you are trying to refer to a glossary definition for the term `cloud-controller-manager` but there's no definition for that.

Example - https://kubernetes.io/docs/reference/glossary/?fundamental=true - look for Controller definition in English. This definition is located at `content/en/docs/reference/glossary/controller.md`

For `cloud-controller-manager` glossary definition, you need to define `content/ru/docs/reference/glossary/cloud-controller-manager.md` and for an example you can look at `content/en/docs/reference/glossary/cloud-controller-manager.md`

The second error -

```
ERROR 2021/05/25 11:36:09 [ru] no Resource or Page matching "/static/community-values.md*".
```

This is related to the community values page as the error says. In `content/ru/community/values.md`, you refer to `/static/community-values.md` but there's no such page for Russian language. Example for English language is - `content/en/community/values.md` and the `/static/community-values.md` page is -> `content/en/community/static/community-values.md`. So, for Russian language, you have to create something at `content/ru/community/static/community-values.md`

With that, all the errors should be gone :)

So, in short, you basically need to define these two markdown files -

`content/ru/community/static/community-values.md`

`content/ru/docs/reference/glossary/cloud-controller-manager.md`

Start by creating the above two files as simply empty files and see if `git submodule update --init --recursive --depth 1 && make production-build` works with no errors and then start writing content :)

**Review by jihoon-seo — CHANGES_REQUESTED** [review]
PTAL @mshalmanov !

**Comment by jihoon-seo** [comment]
According to @karuppiah7890 's comment,
@mshalmanov you added `content/ru/docs/reference/glossary/cloud-controller-manager.md` file in the commit e37b083,
but the file `content/ru/community/static/community-values.md` is not added yet.
@mshalmanov Could you PTAL again?
Thanks!

**Review by Arhell — COMMENTED** [review]
/lgtm
/assign @Potapy4

**Comment by Potapy4** [comment]
спасибо!

/approve
